### ISSUE TITLE
NTR: Eosinophilia in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8485]
New term request Eosinophilia in CSF (segal_210125204349): Eosinophils > 10% of leukocyte count in the cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000824
- label: Cerebrospinal fluid eosinophilia
- parents: [CSF pleocytosis (HP:0012229)](https://hpo.jax.org/app/browse/term/HP:0012229)
- pmid: 
- def: Presence of at least 10% eosinophils in the total cerebrospinal fluid (CSF) leukocyte count.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8485
- synonyms: (NONE)|Eosinophilia in the CSF(NONE)
